rules/sha/index.bzl

```diff
@@ -0,0 +1,10 @@
+# Calculates a sha of the contents of the files in `srcs` and outputs it to a file called `name`.txt.
+def sha(name, srcs, **kwargs):
+    native.genrule(
+        name = name,
+        srcs = srcs,
+        outs = [name + ".txt"],
```

Comment:
nit: consider making the file extension `.sum`

Reply:
Done